nextcord/message.py

```diff
@@ -1850,13 +1857,15 @@ async def edit(self, **fields: Any) -> Optional[Message]:
             if content is not None:
                 fields['content'] = str(content)
 
-        try:
-            embed = fields['embed']
-        except KeyError:
-            pass
-        else:
-            if embed is not None:
-                fields['embed'] = embed.to_dict()
+        if 'embed' in fields and 'embeds' in fields:
+            raise InvalidArgument('Cannot pass both embed and embeds parameter to edit()')
+
+        if 'embed' in fields:
+            embed = fields.pop('embed')
+            fields['embeds'] = [embed] if embed is not None else []
```

Comment:
Do you not required `.to_dict()` on this setup? It appears `embeds` and the previous implementation called it.

Reply:
Yeah, I think I messed that up after a few rewrites. It probably does need to be added back.